src/test/java/io/strimzi/kafka/bridge/mqtt/core/ConfigRetrieverTest.java

```diff
@@ -0,0 +1,79 @@
+package io.strimzi.kafka.bridge.mqtt.core;
+
+import io.strimzi.kafka.bridge.mqtt.config.BridgeConfig;
+import io.strimzi.kafka.bridge.mqtt.config.ConfigRetriever;
+import io.strimzi.kafka.bridge.mqtt.config.KafkaConfig;
+import org.junit.Test;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.Map;
+import java.util.Objects;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThrows;
+
+/**
+ * Unit tests for {@link ConfigRetriever}.
+ */
+public class ConfigRetrieverTest {
+
+    /**
+     * Test if the application.properties file has all configuration parameters.
+     */
+    @Test
+    public void testApplicationPropertiesFile() throws IOException {
+        String filePath = Objects.requireNonNull(getClass().getClassLoader().getResource("application.properties")).getPath();
+        Map<String, Object> config = ConfigRetriever.getConfig(filePath);
+        BridgeConfig bridgeConfig = BridgeConfig.fromMap(config);
+
+        // bridge config related tests
+        assertThat("Bridge-ID should be 'my-bridge'",
+                bridgeConfig.getBridgeID(), is("my-bridge"));
+
+        // Mqtt server config related tests
+        assertThat("There should be 2 related mqtt server config parameters",
+                bridgeConfig.getMqttConfig().getConfig().size(), is(2));
+        assertThat("Mqtt server host should be 'localhost'",
+                bridgeConfig.getMqttConfig().getHost(), is("localhost"));
+        assertThat("Mqtt server port should be '1883'",
+                bridgeConfig.getMqttConfig().getPort(), is(1883));
+
+        // Kafka server config related tests
+        assertThat("There should be 2 related kafka config parameters",
+                bridgeConfig.getKafkaConfig().getConfig().size(), is(2));
```

Comment:
Was the application.properties file updated recently? 

This test was failing locally for me, just want to confirm it isn't just an env issue on my side

Reply:
Yes, sorry! I forgot to update this test. 
There is only one configuration parameter for Kafka now. 
We have removed configuration for the producer.

Reply:
it is sorted now.